code/datums/movement/mob.dm

```diff
@@ -222,6 +222,10 @@
 	return T && !((mob_flags & MOB_FLAG_HOLY_BAD) && check_is_holy_turf(T))
 
 /mob/proc/AdjustMovementDirection(var/direction)
+
+	if(!direction || !isnum(direction))
```

Comment:
Feels like https://github.com/NebulaSS13/Nebula/blob/27a6f06844510b7a5dcdc0ae3908c443e73de816/code/datums/movement/movement.dm#L81 should be rejecting attempts to move in invalid directions